bundles/AdminBundle/Controller/Admin/IndexController.php

```diff
@@ -448,6 +450,8 @@ protected function addCustomViewSettings(ViewModel $settings)
 
     public function onKernelController(FilterControllerEvent $event)
     {
+        // enable view auto-rendering
+        $this->setViewAutoRender($event->getRequest(), true, 'twig');
```

Comment:
if you already on that, why don't you return $this->renderTemplate or $this->render on every action instead? Auto View Rendering is not good imo.